pkg/apis/events/v1alpha1/pubsub_validation.go

```diff
@@ -77,11 +77,7 @@ func (current *PubSubSpec) Validate(ctx context.Context) *apis.FieldError {
 	return errs
 }
 
-func (current *PubSub) CheckImmutableFields(ctx context.Context, og apis.Immutable) *apis.FieldError {
-	original, ok := og.(*PubSub)
-	if !ok {
-		return &apis.FieldError{Message: "The provided original was not a PubSub"}
-	}
+func (current *PubSub) CheckImmutableFields(ctx context.Context, original *PubSub) *apis.FieldError {
```

Comment:
Why did this signature change? Seems like it removed compatibility with the `apis.Immutable` interface (see https://github.com/google/knative-gcp/pull/416/files#diff-130541c895e1b10fab4757caecea7ec8L49).

Reply:
`apis.Immutable` is not a thing anymore. It was removed as part of the webhook PR. See [here](https://github.com/knative/pkg/commit/4836f680bbe5dc615e9c822f21c6685d97c6d553).